core/src/main/java/com/homeaway/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -132,44 +138,58 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(StreamRegistr
 
     @Override
     public void run(final StreamRegistryConfiguration configuration, final Environment environment) {
-
         // Step 1 - initialize managed beans
-
         ManagedKStreams managedKStreams = createManagedKStreams(configuration);
-
         ManagedInfraManager managedInfraManager = createManagedInfraManager(configuration);
         Preconditions.checkState(managedInfraManager != null, "managedInfraManager cannot be null.");
-
         ManagedKafkaProducer managedKafkaProducer = createManagedKafkaProducer(configuration);
 
-        // Step 2 - initialize the DAO's
+        // Step 2 - Create the Kafka topics used as DataStore
+        createKafkaTopicUsedAsDatastore(configuration);
 
+        // Step 3 - initialize the DAO's
         initServiceAndDao(configuration, environment, managedKStreams, managedKafkaProducer);
 
-        // Step 3 - initialize and register the SR resources to JerseyEnvironment
-
+        // Step 4 - initialize and register the SR resources to JerseyEnvironment
         initAndRegisterResource(environment);
 
-        // Step 4 - initialize and register monitoring and health check hooks
-
+        // Step 5 - initialize and register monitoring and health check hooks
         environment.getApplicationContext().addServlet(PingServlet.class, "/ping");
         initAndRegisterHealthCheck(configuration, environment, managedKStreams);
 
-        // Step 5 - centralize initialization and configuration of SR server Object Mapper's
+        // Step 6 - centralize initialization and configuration of SR server Object Mapper's
         registerServiceMapper(environment);
 
-        // Step 6 - In order to avoid muddle up with HK2 and Jersey lifecycle dependency
+        // Step 7 - In order to avoid muddle up with HK2 and Jersey lifecycle dependency
         // we wrap up all managed beans in a centralized container to manage server
         // components start and stop ordering
-
         registerManagedContainer(environment, managedKStreams, managedInfraManager, managedKafkaProducer);
 
-        // Step 7 - register list of Jersey Filters.
+        // Step 8 - register list of Jersey Filters.
```

Comment:
Might want to remove `Step nn` numbers so that we don't have to renumber if/when we change steps or move steps around

Reply:
@prabhakar1983 , please remove Step nn numbers